### PR TITLE
Add Programmer's Notepad

### PR DESCRIPTION
**Pull request by chirsz-ever**

### CODE REVIEW

**Review by HBelusca — CHANGES_REQUESTED** [review]
Can you show us an example that proves that this application works fine in ReactOS?

**Comment by chirsz-ever** [comment]
VirtualBox_ReactOS-0 4-2010030004

**Review by learn-more — APPROVED** [review]

**Review by HBelusca — APPROVED** [review]

File: pnotepad.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[Section]
+Name = Programmer's Notepad
+Version = 2.4.2
+License = Free Software
+Description = Programmer's Notepad is a text editor for Windows.
+Size = 2.4 MiB
```
Comment (learn-more) [inline pnotepad.txt:6]: Size not needed.

**Comment by learn-more** [comment]
Thanks for your contribution, sorry for the delay!

**Pull request merged**